uicomponent-compose/feed/src/main/java/io/github/droidkaigi/feeder/feed/FeedScreen.kt

```diff
@@ -279,13 +280,15 @@ fun FeedItemRow(
     if (showDivider) {
         Divider()
     }
-    FeedItem(
-        feedItem = item,
-        favorited = favorited,
-        onClick = onClickFeed,
-        showMediaLabel = showMediaLabel,
-        onFavoriteChange = onFavoriteChange
-    )
+    AppThemeWithBackground {
```

Comment:
Thank you!
You're not bad because the issue title was bad, but the intention was to make it look better in Preview.
The implementation image is as follows.

```
@Preview(showBackground = true)
@Composable
fun PreviewFeedScreen() {
    AppThemeWithBackground {
        ProvideFeedViewModel(viewModel = fakeFeedViewModel()) {
```

Reply:
Oh, Sorry I couldnt understand. 🙇 I could understand your description. Thank you 🙇